bin/cdn-upload

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+const fs = require('fs')
+const path = require('path')
+const semver = require('semver')
+const { S3Client, PutObjectCommand } = require('@aws-sdk/client-s3')
+const { CloudFrontClient, CreateInvalidationCommand } = require('@aws-sdk/client-cloudfront')
+
+const packageJson = require(`${__dirname}/../packages/platforms/browser/package.json`)
+const packageName = packageJson.name
+const packageVersion = packageJson.version
+const bucketName = process.env.BUCKET_NAME || 'bugsnagcdn'
+const distributionId = process.env.DISTRIBUTION_ID || 'E205JDPNKONLN7'
+const region = process.env.AWS_REGION || 'us-east-1'
```

Comment:
I'm sure we've already discussed these before - but would it be better to only use the environment variables?

Reply:
Maybe; I think this is safe because you can't have AWS credentials laying around in your environment anymore, so it's always a conscious effort to run this script with the ability to upload to the real CDN

That said, the release script could just set them automatically so I don't think there's a downside